lint-staged.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    "*.ts": [
```

Comment:
Nice! Could you also set "*.js" files as the target for prettier?

Reply:
ESLint shows an error `'module' is not defined. (eslint/no-undef)` on some js files such as `husky.config.js`.
I added these files to `.eslintignore` but it is still formatted by `prettier`.

Reply:
I got it. It sounds good.